### ISSUE TITLE
Getting an error

### ISSUE THREAD

**Issue by eralpkor** [riazXrazor/udemy-dl#9]
Running on Mac OS X 10.12.6
Getting an error
"env: node\r: No such file or directory"

**Comment by chaitanya11** [comment]
Yeah i am also getting same error
`/usr/bin/env: ‘node\r’: No such file or directory`

**Comment by chaitanya11** [comment]
@eralpkor  i had the same error but got resolve like this 
`npm install -g udemy-dl`

this command gives 

```
/usr/local/bin/udl -> /usr/local/lib/node_modules/udemy-dl/index.js
/usr/local/lib
└── udemy-dl@0.0.10 
```

then install `dos2unix`
like this 

`brew install dos2unix`

then convert udemy-dl `index.js `file tp unix format like this 

`dos2unix /usr/local/lib/node_modules/udemy-dl/index.js`

then try to run `udl`

**Comment by riazXrazor** [comment]
Thanks I'll hv a look into it, come up with a fix soon.

**Comment by mattwelke** [comment]
Isn't this just a matter of carriage return characters being accidentally left in the source code, making it only runnable on Windows? Fix should be as simple as replacing all instances of ```\r\n``` with ```\n``` in all of the source files. The problem probably came from proper Git configuration from one of the devs, with them changing it to commit Windows-style. Committing UNIX-style is cross-platform.

**Comment by riazXrazor** [comment]
i hv added .gitattributes file i think this will fix the issue

**Comment by zhiyuxia** [comment]
Running on Mac OS X 10.11.6
Getting an error
"env: node\r: No such file or directory"

**Comment by danielbintar** [comment]
Running on Ubuntu 
Getting an error
/usr/bin/env: ‘node\r’: No such file or directory

**Comment by JosimarCamargo** [comment]
It was happening the same error with me, and then I update my node to v8.9.1 and then the error 
"env: node\r: No such file or directory" is not happening more, the application starts well, asks me email,  password and url, next to checking course url, I am getting this error:

Checking course url...                   /home/josimar/.nvm/versions/node/v8.9.1/lib/node_modules/udemy-dl/lib/core.js:108
        callback(b.match(/data-course-id="(\d+)"/)[1]);
                                                  ^

TypeError: Cannot read property '1' of null
    at Request._callback (/home/josimar/.nvm/versions/node/v8.9.1/lib/node_modules/udemy-dl/lib/core.js:108:51)
    at Request.self.callback (/home/josimar/.nvm/versions/node/v8.9.1/lib/node_modules/udemy-dl/node_modules/request/request.js:186:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/home/josimar/.nvm/versions/node/v8.9.1/lib/node_modules/udemy-dl/node_modules/request/request.js:1163:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/home/josimar/.nvm/versions/node/v8.9.1/lib/node_modules/udemy-dl/node_modules/request/request.js:1085:12)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)

**Comment by rmemon** [comment]
Hi @riazXrazor 
I am facing same issue on ubuntu system,

**/usr/bin/env: ‘node\r’: No such file or directory**

can one can provide solution for the same

**Comment by riazXrazor** [comment]
this issue has been fixed, plz update to 0.0.13
